platform-hub-web/src/app/home/home.module.js

```diff
@@ -2,7 +2,7 @@ import angular from 'angular';
 
 import {HomeComponent} from './home.component';
 
-// Define the app section name
+// Main section component name
 export const AppHome = 'appHome';
 
```

Comment:
This is probably just my lack of angular knowledge but I'm not 100% sure what this file is for. You'll have to teach me this bit